bin/swagger_serve_doc

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+swagger generate spec --base-path=./cmd/mysterium_client/ -o ./tequilapi_spec.json --scan-models
+swagger serve ./tequilapi_spec.json -F=swagger
```

Comment:
Instead of `brew`, we can download `swagger` from sources, adding it as glide dependency:
```bash
go get -u github.com/go-swagger/go-swagger/cmd/swagger
```